lib/epoch.ex

```diff
@@ -1,27 +1,40 @@
 defmodule DateTimeParser.Epoch do
   @moduledoc false
 
-  import NimbleParsec
-  import DateTimeParser.Formatters, only: [format: 1]
-  import DateTimeParser.Combinators.Epoch
+  @token_key :unix_epoch
```

Comment:
I'm interested in what you think about pulling this out into a module attribute

Reply:
this is fine with me! you're the artist here and it looks like good art to me.